sql/core/src/test/scala/org/apache/spark/sql/JoinSuite.scala

```diff
@@ -50,8 +49,9 @@ class JoinSuite extends QueryTest with SharedSQLContext {
     assert(planned.size === 1)
   }
 
-  def assertJoin(pair: (String, Class[_])): Any = {
-    val (sqlString, c) = pair
```

Comment:
ditto (let me mark all similar instances in case they are changeable so that you can easily find)